### ISSUE TITLE
[HOLD for payment 2024-01-25] [Card Settings] 🐛 Damaged flow is saying it will immediately cancel the card, rather than saying it will continue to work until the new card is activated

### ISSUE THREAD

**Issue by kevinksullivan** [Expensify/App#33514]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

Version Number: v1.4.16-3
Reproducible in staging?: Y
Reproducible in production?: Y
Email or phone of affected tester (no customers): [kevin+42@expensifail.com](mailto:kevin+42@expensifail.com)
**Issue reported by:** @kevinksullivan 
**Slack conversation:** https://expensify.slack.com/archives/C05DWUDHVK7/p1703265858599339

## Action Performed:
1. Navigated to wallet
2. Tapped Expensify Card row
3. Selected Report physical card loss / damage
4. Selected My card was damaged

## Expected Result:
The copy should explain that the damaged card will remain active until the new one is activated button should say `Ship card`

## Actual Result:
We're showing the same copy / button as we do for a stolen card, which is a different flow that immediately cancels the card and ships a new card with a different PAN

## Screenshots/Videos

![image](https://github.com/Expensify/App/assets/22508290/648dbf9d-d67d-4247-948b-9fbf0ad438f3)

![image](https://github.com/Expensify/App/assets/22508290/9bcf6e47-bec2-4246-a1b5-8271fc70f114)

**Comment by kevinksullivan** [comment]
CC @JKobrynski can you comment so I can assign?

**Comment by grgia** [comment]
[This is the relevant file (https://github.com/Expensify/App/blob/9208c95e3c9b375897e7133696ffcdb07b67fb74/src/pages/settings/Wallet/ReportCardLostPage.js#L80)for tracking

**What needs to be changed:**
There are two flows here depending on the first choice:  Lost/Stolen OR Damaged. Both flows will still make the same API call.

**LOST/STOLEN**
![image](https://github.com/Expensify/App/assets/38015950/ea74f2a9-4534-4334-b5e0-946b84d1730e)

- copy under address confirmation reads: "Your current card will be permanently deactivated as soon as your order is placed. Most cards arrive in a few business days."
- button copy reads: "Deactivate card"
- after pressing button, the current physical card should be in DEACTIVATED state and the new physical card should be in NOT ACTIVATED state

**DAMAGED**
![image](https://github.com/Expensify/App/assets/38015950/e59f91da-1ff2-496c-892a-ba4e19a68af1)

- copy under address confirmation reads: "Your new card will arrive in 2-3 business days, and your existing card will continue to work until you activate your new one."
- button copy reads: "Ship new card"
- after pressing button, the current physical card should be in ACTIVE state (3) and the new physical card should be in NOT ACTIVATED state

**Comment by grgia** [comment]
bumped here https://expensify.slack.com/archives/C03UK30EA1Z/p1704227935701489?thread_ts=1703268549.160679&cid=C03UK30EA1Z

**Comment by grgia** [comment]
bumped in slack, should get this assigned out soon!

**Comment by grgia** [comment]
If not, I'll make this internal

**Comment by JKobrynski** [comment]
Hi, I'm Julian from Callstack - expert contributor group - and I would like to take a look at this issue.

**Comment by JKobrynski** [comment]
The [PR](https://github.com/Expensify/App/pull/34098) is ready for review!

**Comment by kevinksullivan** [comment]
Looks like we're waiting on @grgia for review here!

**Comment by kevinksullivan** [comment]
@hoangzinh offer sent, lmk when you accept

https://www.upwork.com/jobs/~017041742568c125cc

**Comment by kevinksullivan** [comment]
All set
